src/smspark/bootstrapper.py

```diff
@@ -428,7 +428,7 @@ def get_yarn_spark_resource_config(
                 "spark.executor.instances": f"{executor_count_total}",
                 "spark.default.parallelism": f"{default_parallelism}",
                 "spark.yarn.appMasterEnv.AWS_REGION": f"{aws_region}",
-                "spark.executorEnv.AWS_REGION": f"{aws_region}"
+                "spark.executorEnv.AWS_REGION": f"{aws_region},"
```

Comment:
is it expected that comma is within "?

Reply:
No, thanks for pointing out, fixed in the next revision